x/interchainstaking/keeper/receipt.go

```diff
@@ -10,22 +10,24 @@ import (
 	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/x/authz"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
-
 	icatypes "github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/types"
 	transfertypes "github.com/cosmos/ibc-go/v6/modules/apps/transfer/types"
 	clienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
 	channeltypes "github.com/cosmos/ibc-go/v6/modules/core/04-channel/types"
+	prtypes "github.com/quicksilver-zone/quicksilver/x/participationrewards/types"
 
 	"github.com/quicksilver-zone/quicksilver/utils/addressutils"
 	"github.com/quicksilver-zone/quicksilver/x/interchainstaking/types"
 	minttypes "github.com/quicksilver-zone/quicksilver/x/mint/types"
 )
 
 const (
-	Unset           = "unset"
-	ICAMsgChunkSize = 5
-	ICATimeout      = time.Hour * 6
+	Unset                 = "unset"
+	ICAMsgChunkSize       = 5
+	ICATimeout            = time.Hour * 6
+	AuthzAutoClaimAddress = "quick1psevptdp90jad76zt9y9x2nga686hutgmasmwd"
```

Comment:
This will be different between dev/test/prod so will need to be stored as a parameter against the module.

Reply:
@joe-bowman yeah but I see it is hardcoded in the [ui code](https://github.com/quicksilver-zone/quicksilver/blob/main/web-ui/components/Assets/rewardsClaim.tsx#L40)

Reply:
There are a bunch of things hardcoded in the ui that ought not to be :)

Reply:
I propose that the front end be updated to pull from this value as the source of truth

Reply:
@joe-bowman then how should we add the authz address sir, could it be via governance or something else?

Reply:
If we add to the param set, then it will already be updatable by gov